frontend/angular-js/README.md

```diff
@@ -10,11 +10,11 @@
 
 ## Project Bootstrapper
 
-At [Wolox](http://wolox.com.ar), we have developed the [Angular Bootstrap](http://github.com/wolox/frontend-bootstrap) (use `angular` branch) to create our Angular projects. You'll find a detailed guide to start with your project in its README.
+At [Wolox](http://wolox.com.ar), we have developed the [AngularJS Bootstrap](http://github.com/wolox/frontend-bootstrap) (use `angular` branch) to create our AngularJS projects. You'll find a detailed guide to start with your project in its README.
```

Comment:
This is no longer the oficial angularjs bootstrap... This is the url of the new one
https://github.com/Wolox/angularjs-bootstrap

Reply:
I see! I'll change it